_sips/sips/trailing-commas.md

```diff
@@ -1,5 +1,5 @@
 ---
-layout: inner-page-no-masthead
+layout: sip
```

Comment:
That change needs to be applied here: https://github.com/scala/improvement-proposals/blob/main/content/trailing-commas.md

Reply:
see https://github.com/scala/improvement-proposals/pull/52